main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import "fmt"
+
+func worker(id int, jobs <-chan int, results chan<- int) {
```

Comment:
if we are going to return a bunch of results, we need to use a buffered channel or an global slice with `sync.Mutex`